src/scripts/lightgbm_python/score.py

```diff
@@ -42,6 +42,8 @@ def get_arg_parser(parser=None):
     group_i = parser.add_argument_group("Input Data")
     group_i.add_argument("--data",
         required=True, type=input_file_path, help="Inferencing data location (file path)")
+    group_i.add_argument("--nthreads",
```

Comment:
can you roll this change back? I believe this was intended for the treelite score.py